ingestion_server/ingestion_server/ingest.py

```diff
@@ -298,7 +306,15 @@ def reload_upstream(table, progress=None, finish_time=None, approach="advanced")
 
         # Step 3: Import data into a temporary table
         log.info("Copying upstream data...")
-        copy_data = get_copy_data_query(table, shared_cols, approach=approach)
+        environment = config("ENVIRONMENT", default="local").lower()
+        limit_default = 100_000
+        if environment in {"prod", "production"}:
```

Comment:
We should settle on one of these in the future.

Reply:
Agreed, just trying to cover all my bases :sweat_smile: We've had some discussion about `local` as well in #475, I'll make an issue for unifying this everywhere.